data-manipulation/encryption/encrypt-or-decrypt-via-wincrypt.yml

```diff
@@ -10,10 +10,11 @@ rule:
       - A45E377DBB98A6B44FD4034BC3FFF9B0:0x4017A0
   features:
     - and:
-      - or:
-        - api: CryptAcquireContext
-        - api: CryptGenKey
-        - api: CryptImportKey
       - or:
         - api: CryptEncrypt
         - api: CryptDecrypt
+      - optional:
+        - or:
+          - api: CryptAcquireContext
+          - api: CryptGenKey
+          - api: CryptImportKey
```

Comment:
Making this block optional so the rule can match samples that call these APIs in a different function.

Reply:
nice, i encountered the same issue this week